### DISCUSSION TITLE
Custom wazuh decoder

### DISCUSSION THREAD

**Discussion by srijankafle** [Security-Onion-Solutions/securityonion#4347] [Unsupported Versions]
Hi team,

I have a custom decoder written for meraki firewall which has been added to /nsm/wazuh/etc/decoders/local_decoder.xml. I have tested the decoder in wazuh platform installed in independent machine (version 4.1). Similarly, I have also added the rules at /nsm/wazuh/etc/rules/local_rules.xml

I am not able to check if the decoder is working in the platform as I am not able to find the ossec-logtest/wazuh-logtest tool inside the bin folder of /var/ossec. The incoming logs do not seem to be decoded while exploring in Kibana platform. I have restarted the required services after changing the configuration. Am I missing something. Aplologies for mistakes as I come from wazuh platform and am new to SO.

I have attached a screenshot for the sample of local_decoder and local_rules here
Rules:
![image](https://user-images.githubusercontent.com/60125868/120115183-4287f080-c1a2-11eb-8ffa-a94cb63a7c3a.png)

Decoder:
![image](https://user-images.githubusercontent.com/60125868/120115203-54699380-c1a2-11eb-8a0b-23578f893e6f.png)

**Comment by weslambert** [comment]
Try the following to see if it works for you:

`docker exec -it so-wazuh /var/ossec/bin/ossec-logtest`
